vtex/omnishipping
# Fixes sending address wrongly with validation to API

#### What is the purpose of this pull request?
Fixes sending address wrongly with validation to API

#### What problem is this solving?
Closes [story](https://app.clubhouse.io/vtex/story/22971/request-com-formato-de-address-id-incorreto)

#### How should this be manually tested?
1. Add [items to cart](https://fernando--qastore.vtexcommercebeta.com.br/checkout/cart/add?sku=110&qty=1&seller=1&sku=350&qty=1&seller=1&sku=112&qty=1&seller=1&sku=113&qty=1&seller=1&sku=288&qty=1&seller=1&sc=1)
2. Identify as novouseracre@mailinator.com
3. Simulations should send the correct `addressId` not giving errors

#### Screenshots or example usage
n/a
#### Types of changes
